### PR TITLE
video conference

### PR DESCRIPTION
**Pull request by lastpeony**
https://github.com/ant-media/ant-media-documentation/issues/358

### CODE REVIEW

**Comment by mustafaboleken** [comment]
We can mention about the mesh vs mcu vs sfu architectures and say that we are using sfu method in the introduction part

File: docs/guides/conference/video-conference/video-conference.md [hunk 1]
```diff
@@ -0,0 +1,1861 @@
...
+```https://{ams-url}:5443/{app-name}/rest/v2/broadcasts/{participant-stream-id}```
+
+Example:
+```https://test.antmedia.io:5443/LiveApp/rest/v2/broadcasts/idiTofPCrEx4```
+
+Observe that mainTrackId is set to our rooms id.
+
+![](@site/static/img/conference/video-conference/video-conference-4.png)
+
+## Develope a Video Conference Application In React
```
[the remark this answers is not in the document]
Comment (lastpeony) [reply docs/guides/conference/video-conference/video-conference.md]: done

**Comment by lastpeony** [comment]
> We can mention about the mesh vs mcu vs sfu architectures and say that we are using sfu method in the introduction part

not sure about usefulness of this. i need more eloboration

**Comment by lastpeony** [comment]
Sent room security commit @mustafaboleken @yashtandon113

**Review by yashtandon113 — APPROVED** [review]

**Pull request merged**